qlkube/src/decorateSubscription.js

```diff
@@ -147,7 +327,13 @@ function decorateSubscription(baseSchema, targetType, enumType, kubeApiUrl) {
     payload: ${targetType}
   }
     ${subType} {
-      ${subscriptionField}(name: String, namespace: String!): ${subscriptionType}
+      ${subscriptionField}(
+        name: ${
+          resourceApiMainType.resource === 'tenants' ? 'String!' : 'String'
```

Comment:
Would make a variable earlier in order to make this cleaner

Reply:
like: 

`const isTenant = resourceApiMainType.resource === 'tenants'` 

and then:

```
name: ${
          isTenant  ? 'String!' : 'String'
```

or somethings like:

`nameType = resourceApiMainType.resource === 'tenants' ? 'String!' : 'String';`

and then:

`name: ${nameType}`

Reply:
the second you wrote